pkg/analytics/fields_listener.go

```diff
@@ -44,7 +44,14 @@ func (fl *fieldListener) Write(entry zapcore.Entry, fields []zapcore.Field) erro
 	allFields = append(allFields, fl.fields...)
 	allFields = append(allFields, fields...)
 
-	safeLogsStr := logging.SanitizeFields(allFields, fl.client.Hash)
+	var safeLogsStr string
+
+	if len(allFields) != 0 {
+		safeLogsStr = logging.SanitizeFields(allFields, fl.client.Hash)
+	} else {
+		safeLogsStr = entry.Message
```

Comment:
I feel strongly that we should not do this. This basically turns sanitization into an opt-out process; I feel it should always be opt-in. It's way too easy to drop sensitive info into the Sprintf of a log message, and way too hard to remember (either when writing or reviewing) that it would end up on our server.

For example, yesterday Jordan put up a PR that added error logging that includes the value of `--app`. That's a totally appropriate log message, and it provides a good local experience; but it's sensitive data, and we shouldn't send it to  our server.

Reply:
We had a convo about this and pivoted to using a field to indicate sending the message w/o sanitizing